packages/grid_client/src/primitives/nodes.ts

```diff
@@ -332,6 +335,8 @@ class Nodes {
       status: "up",
       page: options.page,
       size: options.size,
+      has_gpu: options.hasGPU,
+      rented_by: options.rentedBy,
```

Comment:
Why underscores?

Reply:
cause this is how the query looks like in the gridproxy